samples/android/Continote/app/src/main/java/com/firebasecontinue/sample/continote/MyNotesActivity.java

```diff
@@ -69,40 +198,176 @@ protected void handleUserSignedOut() {
     }
 
     /**
-     * Handles when the user taps the testFirebaseContinueButton.
+     * Opens the Edit Note screen for the Note with the provided database key.
      *
-     * TODO: Remove this and related library testing code when this sample is more fleshed out.
+     * @param databaseKey The Firebase Realtime Database key of the Note to edit.
+     */
+    private void openEditScreenForNoteWithKey(@NonNull final String databaseKey) {
+        if (databaseKey == null || databaseKey.length() == 0) {
```

Comment:
Done (here and elsewhere, using https://developer.android.com/reference/android/text/TextUtils.html#isEmpty(java.lang.CharSequence).

Neat, thanks!